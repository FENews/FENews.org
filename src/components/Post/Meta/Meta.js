import React from 'react';
import moment from 'moment';
import styles from './Meta.module.scss';

const Meta = ({ date }) => (
  <div className={styles['meta']}>
    <p className={styles['meta__date']}>Published {moment(date).format('D MMM YYYY')}</p>
  </div>
);

export default Meta;
