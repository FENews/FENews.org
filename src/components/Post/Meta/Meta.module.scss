@import '../../../assets/scss/variables';
@import '../../../assets/scss/mixins';

.meta {
  &__date {
    font-style: italic;
  }

}