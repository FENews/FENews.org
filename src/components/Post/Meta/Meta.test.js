import React from 'react';
import renderer from 'react-test-renderer';
import Meta from './Meta';

describe('Meta', () => {
  it('renders correctly', () => {
    const props = {
      date: '2016-09-01'
    };

    const tree = renderer.create(<Meta {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
