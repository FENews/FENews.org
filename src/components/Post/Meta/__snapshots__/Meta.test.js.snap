// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meta renders correctly 1`] = `
<div
  className="meta"
>
  <p
    className="meta__date"
  >
    Published 
    1 Sep 2016
  </p>
</div>
`;
